src/ch/imedias/rsccfx/view/RsccView.java

```diff
@@ -1,26 +1,31 @@
 package ch.imedias.rsccfx.view;
 
 import ch.imedias.rsccfx.model.Rscc;
-import javafx.scene.Parent;
 import javafx.scene.layout.BorderPane;
 
 public class RsccView extends BorderPane { // change Parent to GridPane etc.
   private final Rscc model;
-  final ShowTokenView showToken;
+  final RsccShowTokenView showTokenView;
+  final RsccEnterTokenView enterTokenView;
   //declare all elements here
 
-  /** Javadoc comment here. */
-  public RsccView(Rscc model) {
+  // FIXME: Use this as main View and base all of the views around it on this
```

Comment:
@martinfrancois seems like it makes sense that I'll show you what we did today and accomplished. Simply to make sure that we are on the right track.